config/routes.rb

```diff
@@ -10,4 +10,12 @@
   get 'holds/result', to: 'holds#result', as: :result
 
   get '/logout', to: 'sessions#destroy', as: :logout
+
+  # error pages
+  match '/404' => 'errors#not_found', via: :all
+  match '/422' => 'errors#not_found', via: :all
+  match '/500' => 'errors#internal_server_error', via: :all
+
+  # catchall for not predefined requests - keep this at the very bottom of the routes file
+  match '*catch_unknown_routes' => 'errors#not_found', via: :all
```

Comment:
Avoids hash rocket (surprised that rubo didn't pick this up) and uses more recent `to:` syntax (guessing it's more recent)

```suggestion
  # error pages
  match '404', to: 'errors#not_found', via: :all
  match '422', to: 'errors#not_found', via: :all
  match '500', to: 'errors#internal_server_error', via: :all

  # catchall for not predefined requests - keep this at the very bottom of the routes file
  match '*catch_unknown_routes', to: 'errors#not_found', via: :all
```